CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+# [6.1]
+
+-   **New feature** `Dropdown` component now supports a `closeOnMenuItemClick` boolean prop that specifies
+    whether `Dropdown` should close after a menu item is clicked on (assuming `menuItems` has been passed in).
+    Hopefully this reduces overhead and the necessity to implement `manualToggle` and manage `isActive` to achieve
+    this behavior.
```

Comment:
nitpick for changelogs: 

should be `**NEW FEATURE**`

You should also add a breaking change noting that `closeOnBodyClick` is now removed. This will cause all instances of Dropdown in all clients to break. We can fix this in pro-web, but other consumers need to know about it as well.